src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { DataService } from '../../services/DataService'
+import DatePicker from 'react-datepicker'
+import { expensesUrl } from '../../static/apiconfig'
+
+import "react-datepicker/dist/react-datepicker.css"
+
+import * as Styled from './style';
+
+const  ExpenseForm = () => {
+  const [cost, setCost] = useState('')
+  const [category, setCategory] = useState('')
```

Comment:
why mot in single line? `const [cost, category, setCategory, setCost] = useState('')`?

Reply:
We can't do that each state need to be adde individualy